weave-js/src/assets/icons/icon-remove-alt.svg

```diff
@@ -0,0 +1,4 @@
+<svg width="24" height="24" viewBox="0 0 24 24" fill="none" xmlns="http://www.w3.org/2000/svg">
+<path d="M7.25 12C7.25 11.5858 7.58579 11.25 8 11.25H16C16.4142 11.25 16.75 11.5858 16.75 12C16.75 12.4142 16.4142 12.75 16 12.75H8C7.58579 12.75 7.25 12.4142 7.25 12Z" fill="#2B3038"/>
```

Comment:
This hardcoded color value should be replaced with `currentColor` - the addicon script can do this for you if you set that up: https://github.com/wandb/devtools/blob/main/docs/addicon.md

Reply:
I had trouble setting up poetry I think my pyenv and python installs are messed up, had @ibindlish help me run it instead.